go/src/socialapi/workers/payment/api/customer_test.go

```diff
@@ -197,3 +197,37 @@ func TestInfoPlan(t *testing.T) {
 		})
 	})
 }
+
+func TestCreateCustomCustomer(t *testing.T) {
+	Convey("Given a user", t, func() {
+		withTestServer(t, func(endpoint string) {
+			withTestCreditCardToken(func(token string) {
+				Convey("We should be able to create a custom user", func() {
+					cp := &stripe.CustomerParams{
```

Comment:
@usirin New endpoint accepts the arguments in this form
The URL is `koding.com/api/social/payment/custom-customer/create`

Reply:
💯